cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,9 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and image header size for bootloader support
+export IMAGE_ALIGN = 8
+export IMAGE_HDR_SIZE = 512
```

Comment:
Yes, they're CPU dependent, at least the `IMAGE_ALIGN` variable. I think at the beginning I had them in the cpu folder but for some reason I moved them here. I can move them again to the CPU, however I'm not sure if `IMAGE_HDR_SIZE` depends on something. This variable is used by imgtool to add the right offset (512B in this case) at the beginning of the image, and it's always either 256B or 512B aligned, regardless of the headers size. It means however that if for some reason more than  512B of headers is needed, then this variable must be defined for 1024B, therefore, I think we cannot assume it's constant for all situations.

Besides, as I said in a previous comment, there's the possibility to relocate the VTOR in RAM, which allows to define the headers size to an "arbitrary" value (32B in the case of mynewt which locates VTOR on RAM). The 512B alignment is only needed when locating the VTOR on ROM, so the PC can jump to the defined address.